yodo1/rabbitmq/http_client.py

```diff
@@ -35,13 +35,18 @@ def __init__(
         self.uri = uri
 
         uri_obj = urlparse(uri)
-        self.host = uri_obj.hostname
+        self.scheme = uri_obj.scheme
```

Comment:
Lets add a parser here, like `self.scheme = self.scheme.replace("amqp", "http")` so that it will convert both `amqp` and `amqps` headers to `http` and `https`.